### DISCUSSION TITLE
zotero7链接按钮不出跳转信息 只有空白的“链接精灵”页面

### DISCUSSION THREAD

**Discussion by sqWa** [windingwind/zotero-better-notes#978] [Q&A] [answered]
请问Zotero7安装了1.1.4-beta.72版本的zotero-better-notes后，链接按钮显示如下界面，是什么原因？
![1716021725128](https://github.com/windingwind/zotero-better-notes/assets/63779655/374f7ac7-9698-4c81-b26b-fdf5fb3ba5eb)

**Comment by lizihi** [comment]
同问，一样的问题。

**Comment by windingwind** [comment, marked as answer]
已修复。明显是bug的问题反馈请提issue哦

**Reply by sqWa** [reply]
感谢！！

**Reply by lizihi** [reply]
太牛啦，谢谢！
